jobs/promptLicenceCreation.ts

```diff
@@ -110,21 +104,39 @@ const notifyComOfUpcomingReleases = async (emailGroups: EmailContact[]) => {
 }
 /* eslint-enable */
 
+const excludeCasesNotAssignedToPpWithinPast7Days = (caseload: ManagedCase[]): ManagedCase[] => {
+  const previousWeekStart = startOfISOWeek(subDays(new Date(), 7))
+  const previousWeekEnd = endOfISOWeek(subDays(new Date(), 7))
+
+  const isWithinPastWeek = (c: ManagedCase) => {
+    const dateAllocatedToPp = new Date(c.deliusRecord.allocationDate)
+    return dateAllocatedToPp >= previousWeekStart && dateAllocatedToPp <= previousWeekEnd
+  }
+
+  return caseload.filter(isWithinPastWeek)
+}
+
 Promise.all([
   pollPrisonersDueForLicence(
-    startOfWeek(add(new Date(), { weeks: 12 }), { weekStartsOn: 1 }),
-    endOfWeek(add(new Date(), { weeks: 12 }), { weekStartsOn: 1 }),
-    [LicenceStatus.NOT_STARTED],
-    'No licences that are NOT_STARTED'
+    startOfISOWeek(add(new Date(), { weeks: 12 })),
+    endOfISOWeek(add(new Date(), { weeks: 12 })),
+    [LicenceStatus.NOT_STARTED]
   ),
+
   pollPrisonersDueForLicence(
-    startOfWeek(new Date(), { weekStartsOn: 1 }),
-    endOfWeek(add(new Date(), { weeks: 3 }), { weekStartsOn: 1 }),
-    [LicenceStatus.NOT_STARTED, LicenceStatus.IN_PROGRESS],
-    'No licences that are NOT_STARTED or IN_PROGRESS'
-  ),
+    startOfISOWeek(add(new Date(), { weeks: 4 })),
+    endOfISOWeek(add(new Date(), { weeks: 11 })),
+    [LicenceStatus.NOT_STARTED]
+  ).then(caseload => excludeCasesNotAssignedToPpWithinPast7Days(caseload)),
+
+  pollPrisonersDueForLicence(startOfISOWeek(new Date()), endOfISOWeek(add(new Date(), { weeks: 3 })), [
+    LicenceStatus.NOT_STARTED,
+    LicenceStatus.IN_PROGRESS,
+  ]),
 ])
-  .then(([initialPromptCases, urgentPromptCases]) => buildEmailGroups(initialPromptCases, urgentPromptCases))
+  .then(([week13InitialPromptCases, week5to12InitialPromptCases, urgentPromptCases]) =>
```

Comment:
the email template for week 13 and weeks 5-12 is the same. So the prisoner case list can be added to same email.